v2/lints/cabf_br/lint_ocsp_id_pkix_ocsp_nocheck_ext_not_included_test.go

```diff
@@ -0,0 +1,73 @@
+package cabf_br
+
+/*
+ * ZLint Copyright 2020 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v2/lint"
+	"github.com/zmap/zlint/v2/test"
+)
+
+type TestCase struct {
+	Name           string
+	Filename       string
+	ExpectedResult lint.LintStatus
+}
+
+func RunTest(lintName string, testCases []TestCase, t *testing.T) {
+	t.Helper()
+	for _, tc := range testCases {
+		t.Run(tc.Name, func(t *testing.T) {
+			result := test.TestLint(lintName, tc.Filename)
+			if result.Status != tc.ExpectedResult {
+				t.Errorf("expected result %v was %v", tc.ExpectedResult, result.Status)
+			}
+		})
+	}
+}
+
+func TestOCSPIDPKIXOCSPNocheckExtNotIncluded(t *testing.T) {
+	testCases := []TestCase{
+		{
+			Name:           "SMIME CA Wrong",
```

Comment:
It took me a while to work through "why" this was wrong, by manually inspecting the certs and trying to compare logic. I think it might be helpful here to use Name to highlight what the essential properties are (e.g. in the list I just commented above), so it's easy to work through what the cert does or doesn't cover.

This is, admittedly, easier when they have the PEM data themselves.

Reply:
Solved